### ISSUE TITLE
Markdown image size not working anymore

### ISSUE THREAD

**Issue by M-Mueller** [mattermost/mattermost-server#10355]
#### Summary
Any image size set in markdown seems to be ignored since v5.7 (this feature was added with #4803)

#### Steps to reproduce
```
![Bar](https://raw.githubusercontent.com/M-Mueller/mattermost-poll/master/img/bar.png =450x25)
```

#### Expected behavior
Behaviour in v5.5:
![5 5](https://user-images.githubusercontent.com/584616/53297427-07f2b900-381f-11e9-80d8-7a0ab1ea6ec8.png)

#### Observed behavior (that appears unintentional) 
Behaviour in v5.7:
![5 7](https://user-images.githubusercontent.com/584616/53297437-193bc580-381f-11e9-8e3b-52a6d13f2b24.png)

#### Possible fixes
This can be fixed by removing `style="height: initial;"` from the image. However, I couldn't figure out where this is set in the code.

**Comment by amyblais** [comment]
Hi @M-Mueller,

I'm not able to reproduce this on a server that's on v5.9 branch, so it is probably fixed in the upcoming release - https://community.mattermost.com. Would you be able to test and let me know what you see?

**Comment by M-Mueller** [comment]
Seem like this is already fixed in 5.8, sorry for the inconvenience.
